validator/validator-main.protoascii

```diff
@@ -29,7 +29,7 @@ min_validator_revision_required: 474
 spec_file_revision: 1052
 
 styles_spec_url: "https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/style_pages"
-script_spec_url: "https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml#html-tags"
+script_spec_url: "https://www.ampproject.org/docs/troubleshooting/validation_errors#custom-javascript-is-not-allowed"
```

Comment:
I believe this should be amp.dev instead of www.ampproject.org.

Reply:
https://github.com/ampproject/amphtml/pull/28823